ci/scripts/common.env.sh

```diff
@@ -1,5 +1,5 @@
 export CARGO_TERM_COLOR=always
-export RUSTFLAGS="-D warnings --cfg tokio_unstable"
+export RUSTFLAGS="-D warnings -Ctarget-cpu=native --cfg tokio_unstable"
```

Comment:
We are assuming CI will run on the same machine on which it is build.

Reply:
However, this env var is also used for release build. Maybe this is not good. We want our release build to always have SIMD flags enabled.

Reply:
Maybe for release case it is ok to assume that the build machine has SIMD (as the release build servers e.g. CI servers are running on cloud CPUs which tend to have the SIMD extensions)

Reply:
However, the safer path is really to use a different RUSTFLAGS for release build. In fact, we may want to override the RUSTFLAGS here and just use the .config/cargo.toml profile. This is as simple as specifying `unset RUSTFLAGS` in the release script.

Reply:
done.